src/legacy/core_plugins/embeddable_api/public/index.ts

```diff
@@ -16,46 +16,46 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+export {
```

Comment:
It does look a lot nicer but I was being explicit because everything that is exported from the top level needs to eventually be officially maintained.  Some things in nested folders could be exported for use only internally, not for outside consumption (and then can be updated haphazardly without having to worry about BWC).   

cc @joshdover -- Has this been talked about at all in the context of New Platform?  Is there a recommendation one way or the other?

Reply:
You are selectively exporting on two levels:

- `src/legacy/core_plugins/embeddable_api/public/**/index.ts`
- `src/legacy/core_plugins/embeddable_api/public/index.ts`

Maybe if `embeddable_api/public/actions/index.ts` selectively exports stuff, then on `embeddable_api/public/index.ts` level it could be `export * from './actions.ts'`. Or the other way around.

Reply:
I don't feel strongly one way or the other, but the benefit of being explicit is that I can look one level down and see... okay this is the data I have, what do I want to expose to the outside world?

Reply:
i am pro explicit exports on top level, no strong opinion about lower levels

Reply:
++ on explicit exports at the top level at least